agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/LazyConfigurationAccessor.java

```diff
@@ -69,8 +68,7 @@ public void lazyLoad() {
         System.getenv("APPINSIGHTS_INSTRUMENTATIONKEY"));
     setWebsiteSiteName(System.getenv("WEBSITE_SITE_NAME"));
     setSelfDiagnosticsLevel(System.getenv("APPLICATIONINSIGHTS_SELF_DIAGNOSTICS_LEVEL"));
-    setInstrumentationLoggingLevel(
-        System.getenv("APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL"));
+    // TODO (heya) need to handle APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL
```

Comment:
@heyams this broke when we updated to the real OTel logging instrumentation

Reply:
Need to map severitylevel to loggingLevel for function?

Reply:
Need to call the exporter here to dynamically update the threshold